src/tools/cephfs/tox.ini

```diff
@@ -0,0 +1,8 @@
+[tox]
+envlist = py3 flake8
+skipsdist = true
+
+[testenv:flake8]
+deps = flake8
+passenv = *
```

Comment:
what env variable is flake8 expecting?

Reply:
Previously, I was getting the below warning. I added passenv to fix it.
```
WARNING: Discarding $PYTHONPATH from environment, to override specify PYTHONPATH in 'passenv' in your configuration.
```